rust/src/pd.rs

```diff
@@ -183,4 +178,5 @@ impl Drop for PeripheralDevice {
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
I think we can have a no_std compliant file implementation as well. Was this left out just for simplicity or is there a design problem?

Reply:
I don't know of any way to solve this generally without `std` so I left it out. I think it would make more sense to let the user handle that themselves in the command callback.